kubectl-rook-ceph.sh

```diff
@@ -316,6 +317,7 @@ function run_main_command () {
 # Default values
 : "${ROOK_CLUSTER_NAMESPACE:=rook-ceph}"
 : "${ROOK_OPERATOR_NAMESPACE:=$ROOK_CLUSTER_NAMESPACE}"
+: "${TOP_LEVEL_COMMAND:=kubectl}"
```

Comment:
```suggestion
: "${TOP_LEVEL_COMMAND:=kubectl --namespace "$ROOK_CLUSTER_NAMESPACE"}"
```

Reply:
in some commands, we are using `ROOK_OPERATOR_NAMESPACE`. ex: 
```code
function fetch_mon_endpoints (){
  end_of_command_parsing "$@" # end of command tree
  $TOP_LEVEL_COMMAND --namespace "$ROOK_CLUSTER_NAMESPACE" get cm rook-ceph-mon-endpoints -o json | jq --monochrome-output '.data.data'| tr -d '"' | tr -d '='|sed 's/[A-Za-z]*//g'
}
```